on-screen-keyboard-input/src/KeypadButtons.styled.tsx

```diff
@@ -22,7 +22,7 @@ export const KeypadButtonElement = styled.input`
     background-color: #3653bc;
   }
 `;
-interface IKeypadButtonProps extends InputHTMLAttributes<HTMLInputElement> {}
```

Comment:
changed interface to a type so my typescript config doesn't throw errors.
Of course on a real project, I will update my IDE to align with the project config.